implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -327,6 +326,11 @@ private void send(List<String> metricLines) {
         }
     }
 
+    private String getTruncatedBody(HttpSender.Response response) {
+        return StringUtils.truncate(response.body(), LOG_RESPONSE_BODY_TRUNCATION_LIMIT,
+                LOG_RESPONSE_BODY_TRUNCATION_INDICATOR);
```

Comment:
Now the constants are only used in one single place so you could consider using the values directly instead.

Reply:
@arminru Thanks for the suggestion! I applied it.

Reply:
Sure, thank you! :)